scvi/model/_scvi.py

```diff
@@ -5,7 +5,11 @@
 
 from scvi._compat import Literal
 from scvi._constants import _CONSTANTS
-from scvi.data.anndata._fields import CategoricalObsField, LayerField
+from scvi.data.anndata._fields import (
```

Comment:
we should make this import "public", yes? and then in the next PR we can start building the docs for the new classes

Reply:
sure